### PR TITLE
Make standard nodes variadic

### LINKED ISSUE #1068: Make standard library nodes variadic
### Rationale

OK, we’ve got variadic nodes, let’s use it.

### Prerequisites

#1065, #1067 

### Acceptance criteria

The following nodes are promoted to be variadic:

- [x] xod/bits/bitwise-and
- [x] xod/bits/bitwise-xor
- [x] xod/bits/bitwise-or
- [x] xod/core/any
- [x] xod/core/add
- [x] xod/core/concat
- [x] xod/core/discretize
- [x] xod/core/multiply
- [x] xod/core/nand
- [x] xod/core/nor
- [x] xod/core/select
- [x] xod/core/xor
- [ ] xod/core/nth-number
- [x] xod/core/and
- [x] xod/core/or

### PR DESCRIPTION
**Pull request by evgenykochetkov**
Closes #1068

Also fixes a bug introduced in #1089

### CODE REVIEW

File: packages/xod-project/src/expandVariadicNodes.js [hunk 1]
```diff
@@ -57,8 +57,8 @@ const createAdditionalValueTerminalGroups = (
   const getValueTerminalX = (terminalGroupIndex, terminalIndex) => (
     rightmostInputTerminalX +
     DISTANCE_BETWEEN_TERMINALS +
-    // because terminalGroupIndex always starts from 2
-    ((terminalGroupIndex - 2) * valueTerminalsGroupWidth) +
+    // because terminalGroupIndex always starts from 1
+    ((terminalGroupIndex - 1) * valueTerminalsGroupWidth) +
```
Comment (nkrkv) [inline packages/xod-project/src/expandVariadicNodes.js]: :rofl:

File: workspace/__lib__/xod/bits/bitwise-xor/patch.xodp [hunk 2]
```diff
@@ -1,6 +1,14 @@
 {
-  "description": "Computes bitwise XOR of two bytes. A resulting bit at a particular position is 1 only if either of corresponding bits of inputs is 1 and another is 0.",
+  "description": "Computes bitwise XOR of two or more bytes. A resulting bit at a particular position is 1 only if either of corresponding bits of inputs is 1 and another is 0.",
```
Comment (nkrkv) [inline workspace/__lib__/xod/bits/bitwise-xor/patch.xodp]: A resulting bit at a particular position is 1 only if an odd number of corresponding bits of inputs is 1.

File: workspace/__lib__/xod/core/discretize/patch.xodp [hunk 3]
```diff
@@ -0,0 +1,155 @@
+{
+  "description": "Rounds input `X` up to a nearest threshold — `T0`, `T1`, etc. If `X` is even greater than last threshold, the value is still round to it. Assumes thresholds are in ascending order: `T0` < `T1` etc",
```
Comment (nkrkv) [inline workspace/__lib__/xod/core/discretize/patch.xodp]: Rounds input `X` up to **the first threshold `T0`, `T1`, ..., `T𝑛` greater or equal to the `X`**. If `X` is even greater than **the** last threshold, the value is ~still~ round **down** to it. ~Assumes thresholds are in ascending order: `T0` < `T1` etc~

With changes applied:

Rounds input `X` up to the first threshold `T0`, `T1`, ..., `T𝑛` greater or equal to the `X`. If `X` is even greater than the last threshold, the value is round down to it.

File: workspace/__lib__/xod/core/select/patch.xodp [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "description": "Outputs either `X1` or `X2` value based on which pulse came last — `S1` or `S2`. By default outputs `X1`.",
+  "description": "Outputs `XN` value based on which `SN` pulse came last. By default outputs `X1`.",
```
Comment (nkrkv) [inline workspace/__lib__/xod/core/select/patch.xodp]: Suggest using mathematical 𝑛 character:

Outputs `X𝑛` value based on which `S𝑛` pulse came last. By default outputs `X1`.

**Comment by evgenykochetkov** [comment]
@nkrkv done!

**Pull request merged**